<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChaptersEditor</class>
 <widget class="QDialog" name="ChaptersEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Chapters Editor</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Which one is better?</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QListWidget" name="listLeft"/>
     </item>
     <item row="1" column="1">
      <widget class="QListWidget" name="listRight"/>
     </item>
     <item row="0" column="1">
      <widget class="QRadioButton" name="rdRight">
       <property name="text">
        <string>I like this one</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QRadioButton" name="rdLeft">
       <property name="text">
        <string>I prefer this</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::Retry</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
